package.json

```diff
@@ -43,7 +43,7 @@
     "launchdarkly-js-client-sdk": "^2.22.1",
     "lodash": "^4.17.21",
     "regenerator-runtime": "^0.13.9",
-    "sbc-common-components": "3.0.3",
+    "sbc-common-components": "3.0.4",
```

Comment:
The actual base BaseAddress styling is now in sbc-common-components (CSS override should not longer be needed).